cluster/mesos/cluster.go

```diff
@@ -69,18 +70,18 @@ func NewCluster(scheduler *scheduler.Scheduler, TLSConfig *tls.Config, master st
 		flag.Lookup("logtostderr").Value.Set("true")
 	}
 	cluster := &Cluster{
-		dockerEnginePort:     defaultDockerEnginePort,
-		clusterEventHandlers: cluster.NewClusterEventHandlers(),
-		master:               master,
-		agents:               make(map[string]*agent),
-		TLSConfig:            TLSConfig,
-		options:              &options,
-		offerTimeout:         defaultOfferTimeout,
-		taskCreationTimeout:  defaultTaskCreationTimeout,
-		engineOpts:           engineOptions,
-		refuseTimeout:        defaultRefuseTimeout,
-	}
-
+		dockerEnginePort:    defaultDockerEnginePort,
+		master:              master,
+		agents:              make(map[string]*agent),
+		TLSConfig:           TLSConfig,
+		options:             &options,
+		offerTimeout:        defaultOfferTimeout,
+		taskCreationTimeout: defaultTaskCreationTimeout,
+		engineOpts:          engineOptions,
+		refuseTimeout:       defaultRefuseTimeout,
+	}
+
+	cluster.InitClusterEventHandlers()
```

Comment:
Oh yeah, I had changed it earlier for some reason, but forgot to fix it later. I'll change it.